subsys/fs/CMakeLists.txt

```
@@ -15,7 +15,10 @@ if(CONFIG_FILE_SYSTEM)
   target_link_libraries_ifdef(CONFIG_FAT_FILESYSTEM_ELM   FS INTERFACE ELMFAT)
   target_link_libraries_ifdef(CONFIG_FILE_SYSTEM_LITTLEFS FS INTERFACE LITTLEFS)
 
-  add_partition_manager_config(pm.yml)
+  if (ZEPHYR_NRF_MODULE_DIR)
```

Although we know that `ZEPHYR_NRF_MODULE_DIR` contains this function, then I think it would look nicer to test as:
```suggestion
  if(COMMAND add_partition_manager_config)
```
as that will have no opinion on where this command is defined.

---

If you consider https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/279#issuecomment-591828282 and the conclusion is that we don't need the check anymore, would this comment still be valid?

---

see: https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/279/files/f23c709b956fc782298c494fb77453aa82b5e1f5#r385039160